migration/20171106-rename-default-loan-period.sql

```diff
@@ -0,0 +1 @@
+update collectionsettings set name='ebook_loan_duration' where name='default_loan_period';
```

Comment:
This won't actually do anything because ebook_loan_duration is a per-library per-collection setting, and default_loan_period is a per-collection setting. So I could just delete the old settings.

Reply:
If it matters you could copy the old setting for every library and then delete it. Not sure if anyone is using it.

Reply:
I think I would rather either ignore it or delete it because the old description was vague enough that people didn't know what it did.